core/src/test/java/org/apache/calcite/rex/RexProgramBuilderBase.java

```diff
@@ -306,6 +306,14 @@ protected RexNode gt(RexNode n1, RexNode n2) {
     return rexBuilder.makeCall(SqlStdOperatorTable.GREATER_THAN, n1, n2);
   }
 
+  protected RexNode like(RexNode ref, RexNode pattern) {
+    return rexBuilder.makeCall(SqlStdOperatorTable.LIKE, ref, pattern);
+  }
+
+  protected RexNode like(RexNode ref, RexNode pattern, RexNode escape) {
+    return rexBuilder.makeCall(SqlStdOperatorTable.LIKE, ref, pattern, escape);
+  }
+
```

Comment:
What does this method mean? Could you add some comments?

Reply:
I disagree. There are lots of surrounding methods with a similar pattern. They are protected methods in a test case. Javadoc would not add anything.